readme.md

```diff
@@ -118,6 +106,20 @@ Turn on a specific light.
 hue.turnOnLight("abc123")
 ```
 
+### `hue.turnOnAllLights()`
```

Comment:
Let's consider adding the following methods in the future.

- `turnOnLights(ids)`
- `turnOffLights(ids)`